pom.xml

```diff
@@ -292,6 +346,12 @@ under the License.
       <version>4.11.0</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.hamcrest</groupId>
+      <artifactId>hamcrest-core</artifactId>
+      <version>1.3</version>
```

Comment:
there is a newer version ... as I remember

Reply:
This is what we were getting. I'll look at upgrading dependencies in a separate PR if dependabot doesn't spot it first